backend/models/user.model.ts

```diff
@@ -64,6 +64,7 @@ const UserSchema: Schema = new Schema({
   },
 });
 
+/* eslint-disable func-names */
```

Comment:
https://medium.com/@lucasdavidferrero/dont-use-arrow-functions-when-you-use-mongoose-schema-method-190b79f1640c